### PR TITLE
Mmseqs createdb and linclust container version upgrade + nf-tests

### PR DESCRIPTION
**Pull request by vagkaratzas**
## PR checklist

Closes #XXX 

- [ ] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the module conventions in the [contribution docs](https://github.com/nf-core/modules/tree/master/.github/CONTRIBUTING.md)
- [ ] If necessary, include test data in your PR.
- [ ] Remove all TODO statements.
- [ ] Emit the `versions.yml` file.
- [ ] Follow the naming conventions.
- [ ] Follow the parameters requirements.
- [ ] Follow the input/output options guidelines.
- [ ] Add a resource `label`
- [ ] Use BioConda and BioContainers if possible to fulfil software requirements.
- Ensure that the test works with either Docker / Singularity. Conda CI tests can be quite flaky:
  - [ ] `PROFILE=docker pytest --tag <MODULE> --symlink --keep-workflow-wd --git-aware`
  - [ ] `PROFILE=singularity pytest --tag <MODULE> --symlink --keep-workflow-wd --git-aware`
  - [ ] `PROFILE=conda pytest --tag <MODULE> --symlink --keep-workflow-wd --git-aware`

### CODE REVIEW

**Review by JoseEspinosa — APPROVED** [review]
LGTM! But why are some tests not running? Feel free to merge once they pass

File: modules/nf-core/mmseqs/createdb/tests/main.nf.test [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+    tag "modules"
+    tag "modules_nfcore"
+    tag "mmseqs"
+    tag "mmseqs/createdb"
+
+    test("Should build an mmseqs db from a contigs fasta file") {
+
+        when {
+            params {
+                outdir = "$outputDir"
```
Comment (Joon-Klaps) [inline modules/nf-core/mmseqs/createdb/tests/main.nf.test:15]: Don't need it if you aren't checking for published files.

**Review by Joon-Klaps — CHANGES_REQUESTED** [review]
Awesome 🚀! Small comments, looks like you forgot the `tags.yml` file. 

Thanks as well for fixing some of my typos in the docs.

**Review by Joon-Klaps — APPROVED** [review]
Great work!

File: modules/nf-core/mmseqs/createdb/tests/main.nf.test [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+                ]
+                """
+            }
+        }
+
+        then {
+            assertAll(
+                { assert process.success },
+                { assert snapshot(process.out.db).match("contig_db") },
+                { assert process.out.versions }
```
Comment (sateeshperi) [inline modules/nf-core/mmseqs/createdb/tests/main.nf.test:28]: not including versions in snapshot ? @vagkaratzas @Joon-Klaps

**Pull request merged**